documentation/src/main/asciidoc/chapter-5-data-type-conversions.asciidoc

```diff
@@ -253,6 +253,61 @@ This means that it is possible for MapStruct not to report unmapped target prope
 ====
 
 
+[[invoking-custom-mapping-method]]
+=== Invoking custom mapping method
+
+Sometimes mappings are not straightforward, and some fields should be mapped using plain java code, while mapping latter of them is covered by generated method.
```

Comment:
while mapping latter == while mapping the others

Reply:
Fixed